src/main/java/org/broadinstitute/consent/http/db/mapper/UserWithRolesReducer.java

```diff
@@ -44,6 +46,19 @@ public void accumulate(Map<Integer, User> map, RowView rowView) {
     } catch(MappingException e) {
       //Ignore institution mapping errors, possible for new users to not have an institution
     }
+    //user role join can cause duplication of data if done in tandem with joins on other tables
+    //ex) The same LC can end up being repeated multiple times
+    //Below only adds LC if not currently saved on the array
+    try {
+      if(Objects.nonNull(rowView.getColumn("lc_id", Integer.class))) {
+        LibraryCard lc = rowView.getRow(LibraryCard.class);
+        if(Objects.isNull(user.getLibraryCards()) || !user.getLibraryCards().stream().anyMatch(card -> card.getId() == lc.getId())) {
+          user.addLibraryCard(lc);
+        }
+      }
+    } catch(MappingException e) {
+      //Ignore exceptions here, user may not have a library card issued under this instiution
```

Comment:
Not for this PR, but I think we should start logging these as errors so we can follow up on them.